test/integration/targets/filesystem/tasks/main.yml

```diff
@@ -12,6 +12,7 @@
     grow: '{{ item.value.grow }}'
   when:
     - 'not (item.key == "btrfs" and ansible_system == "FreeBSD")'
+    - 'not (item.key == "ocfs2" and ansible_system != "Ubuntu")'
```

Comment:
Tests are currently not run by the CI because this condition is always `False`, `ansible_distribution` must be used instead of `ansible_system`. Note that `not (item.key == "ocfs2" and ansible_os_family != 'Debian')` could be used too.